pkg/kubectl/cmd/util/factory_client_access.go

```diff
@@ -454,8 +454,8 @@ const (
 	SecretForDockerRegistryV1GeneratorName      = "secret-for-docker-registry/v1"
 	SecretForTLSV1GeneratorName                 = "secret-for-tls/v1"
 	ConfigMapV1GeneratorName                    = "configmap/v1"
-	ClusterRoleBindingV1GeneratorName           = "clusterrolebinding.rbac.authorization.k8s.io/v1alpha1"
```

Comment:
its possible that was not litigated.  my understanding was we used generators to handle when fundamental behavior changed, but api version was always controlled by --output-version .